Bottom: e0a3ef55f02d652d8b18c9b3ab10c455f4271b58
Top:    9c9edc730f86155d4976a7bf8124139b363d12e6
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-11-29 10:44:33 +0100

more primitive types cleanup


---

diff --git a/bytecode/src/main/scala/net/virtualvoid/bytecode/Tools.scala b/bytecode/src/main/scala/net/virtualvoid/bytecode/Tools.scala
index 9ca3c1b..18e3713 100644
--- a/bytecode/src/main/scala/net/virtualvoid/bytecode/Tools.scala
+++ b/bytecode/src/main/scala/net/virtualvoid/bytecode/Tools.scala
@@ -14,6 +14,8 @@ object CodeTools{
     case "scala.Boolean" => java.lang.Boolean.TYPE
     case "double" => java.lang.Double.TYPE
     case "scala.Double" => java.lang.Double.TYPE
+    case "byte" => java.lang.Byte.TYPE
+    case "scala.Byte" => java.lang.Byte.TYPE
     case _ => java.lang.Class.forName(name)
   }
